### ISSUE TITLE
Parser issue

### ISSUE THREAD

**Issue by ghost** [SpiderLabs/ModSecurity#2076]
**Describe the bug**

I'm rewriting my old OWASP ruleset for modsecurity 3.0.3. I encounter an error with one file. After searching the error for a time 

**Logs and dumps**

See below

**To Reproduce**

This config file (called a.conf):

```
SecRule TX:text_file_extension "@eq 1" "chain,phase:3,allow,nolog,id:'999008',severity:'6'"
SecRule TX:no_parameters "@eq 1"
# comment
```

Returns the following error:

Modsecurity msc_rules_add_file("a.conf"): Rules error. File: a.conf. Line: 3. Column: 7. Expecting an action, got:  # comment.

I add a jump line, and I no longer have error.

```
SecRule TX:text_file_extension "@eq 1" "chain,phase:3,allow,nolog,id:'999008',severity:'6'"
SecRule TX:no_parameters "@eq 1"

# comment
```

**Expected behavior**

Not sure, but it seems that one of the two cases was wrong.

Note that this behavior is pernicious because the error appear between file. If a file ends with:

a.conf:
```
SecRule TX:text_file_extension "@eq 1" "chain,phase:3,allow,nolog,id:'999008',severity:'6'"
SecRule TX:no_parameters "@eq 1"
```

And another one starts with:

b.conf
```
# comment
```

And the main file:

```
Include *.conf
```

The error is triggered.

If the main file is:

```
Include a.conf
Include b.conf
```

The error is not trigered.

**Server (please complete the following information):**
 - ModSecurity v3.0.3 with homemade exploitation
 - debian 8

**Rule Set (please complete the following information):**
 - Only the file displayed above

**Comment by victorhora** [comment]
Hi @tech-ozon-io,

Thanks for your detailed report. I think that this issue is already fixed in the current master branch ([3.0.4-pre](https://github.com/SpiderLabs/ModSecurity/blob/v3/master/CHANGES#L1-L55)). Please let us know if the issue persists with a build from a clone from master.

Thanks.

**Comment by zimmerle** [comment]
Based on @victorhora commets I am closing this issue. @tech-ozon-io let us know if that is still a problem.
